packages/analytics-browser-test/test/index.test.ts

```diff
@@ -148,7 +190,6 @@ describe('integration', () => {
         os_name: 'WebKit',
         os_version: '537.36',
         device_manufacturer: undefined,
-        device_model: undefined,
```

Comment:
Why device_model got removed from this file?

Reply:
the expected value does not have it explicitly and jest prefers that it's not specified on the received value. but regardless if i remove or not, the assertion is the "same" so this is mostly clean up.